arangodb/datadog_checks/arangodb/metrics.py

```diff
@@ -0,0 +1,139 @@
+    'arangodb_collection_lock_acquisition_time': 'rocksdb.collection_lock.acquisition_time',
+    'arangodb_rocksdb_write_stalls_total': 'rocksdb.write.stalls',
+    'arangodb_rocksdb_write_stops_total': 'rocksdb.write.stops',
+    'rocksdb_actual_delayed_write_rate': 'rocksdb.actual.delayed.write.rate',
+    'rocksdb_archived_wal_files': 'rocksdb.archived.wal.files',
+    'rocksdb_background_errors': 'rocksdb.background.errors',
+    'rocksdb_base_level': 'rocksdb.base.level',
+    'rocksdb_block_cache_capacity': 'rocksdb.block.cache.capacity',
+    'rocksdb_block_cache_pinned_usage': 'rocksdb.block.cache.pinned.usage',
+    'rocksdb_block_cache_usage': 'rocksdb.block.cache.usage',
+    'rocksdb_cache_hit_rate_lifetime': 'rocksdb.cache.hit.rate.lifetime',
+    'rocksdb_cache_limit': 'rocksdb.cache.limit',
+    'rocksdb_compaction_pending': 'rocksdb.compaction.pending',
+    'rocksdb_cur_size_active_mem_table': 'rocksdb.cur.size.active.mem.table',
+    'rocksdb_cur_size_all_mem_tables': 'rocksdb.cur.size.all.mem.tables',
+    'rocksdb_engine_throttle_bps': 'rocksdb.engine.throttle.bps',
+    'rocksdb_estimate_live_data_size': 'rocksdb.estimate.live.data.size',
+    'rocksdb_estimate_num_keys': 'rocksdb.estimate.num.keys',
+    'rocksdb_estimate_pending_compaction_bytes': 'rocksdb.estimate.pending.compaction.bytes',
+    'rocksdb_estimate_table_readers_mem': 'rocksdb.estimate.table.readers.mem',
+    'rocksdb_free_disk_space': 'rocksdb.free.disk.space',
+    'rocksdb_free_inodes': 'rocksdb.free.inodes',
+    'rocksdb_live_sst_files_size': 'rocksdb.live.sst.files.size',
+    'rocksdb_mem_table_flush_pending': 'rocksdb.mem.table.flush.pending',
+    'rocksdb_min_log_number_to_keep': 'rocksdb.min.log.number.to.keep',
+    'rocksdb_num_deletes_active_mem_table': 'rocksdb.num.deletes.active.mem.table',
+    'rocksdb_num_deletes_imm_mem_tables': 'rocksdb.num.deletes.imm.mem.tables',
+    'rocksdb_num_entries_active_mem_table': 'rocksdb.num.entries.active.mem.table',
+    'rocksdb_num_entries_imm_mem_tables': 'rocksdb.num.entries.imm_mem.tables',
+    'rocksdb_num_immutable_mem_table': 'rocksdb.num.immutable.mem.table',
+    'rocksdb_num_immutable_mem_table_flushed': 'rocksdb.num.immutable.mem.table.flushed',
+    'rocksdb_num_live_versions': 'rocksdb.num.live.versions',
+    'rocksdb_num_running_compactions': 'rocksdb.num.running.compactions',
+    'rocksdb_num_running_flushes': 'rocksdb.num.running.flushes',
+    'rocksdb_num_snapshots': 'rocksdb.num.snapshots',
+    'rocksdb_prunable_wal_files': 'rocksdb.prunable.wal.files',
+    'rocksdb_size_all_mem_tables': 'rocksdb.size.all.mem.tables',
+    'rocksdb_total_disk_space': 'rocksdb.total.disk.space',
+    # health:
+    'arangodb_dropped_followers_total': 'health.dropped_followers',
+    'arangodb_heartbeat_failures_total': 'health.heartbeat_failures',
+    'arangodb_heartbeat_send_time_msec': 'health.heartbeat.sent.time',
+}
+
+
+def construct_metrics_config(metric_map, type_overrides):
```

Comment:
Only need this for checks supporting legacy v1 OpenMetrics implementation; do the right thing in the defined mapping